dask_cuda/benchmarks/local_cudf_shuffle.py

```diff
@@ -63,12 +51,20 @@ def run(client, args, n_workers, write_profile=None):
     wait(df)
     data_processed = len(df) * sum([t.itemsize for t in df.dtypes])
 
-    if args.backend == "dask":
-        took = shuffle_dask(args, df, write_profile)
+    if write_profile is None:
+        ctx = contextlib.nullcontext()
     else:
-        took = shuffle_explicit_comms(args, df)
+        ctx = performance_report(filename=args.profile)
+
+    with ctx:
+        t1 = clock()
```

Comment:
JFYI we may want to replace this with `perf_counter` (or similar) in the future, but that's unrelated to this change

Reply:
NVM this is done [above]( https://github.com/rapidsai/dask-cuda/blob/81bbc6f85575826b13b3fb45894b54135514e668/dask_cuda/benchmarks/local_cudf_shuffle.py#L3 )